### PR TITLE
Sync cloud 3.3

### PR DESCRIPTION
**Pull request by Gaya**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: graylog2-web-interface/src/routing/AppRouter.jsx [hunk 1]
```diff
@@ -211,13 +214,17 @@ const AppRouter = () => {
...
+                        {!isCloud && <Route path={Routes.SYSTEM.INDEX_SETS.CREATE} component={IndexSetCreationPage} />}
+                        {!isCloud && (
+                          <Route path={Routes.SYSTEM.INDEX_SETS.SHOW(':indexSetId')} component={IndexSetPage} />
+                        )}
+                        {!isCloud && (
+                          <Route path={Routes.SYSTEM.INDEX_SETS.CONFIGURATION(':indexSetId')}
+                                 component={IndexSetConfigurationPage} />
+                        )}
+                        {!isCloud && <Route path={Routes.SYSTEM.INDICES.LIST} component={IndicesPage} />}
+                        {!isCloud && <Route path={Routes.SYSTEM.INDICES.FAILURES} component={IndexerFailuresPage} />}
```
Comment (edmundoa) [inline graylog2-web-interface/src/routing/AppRouter.jsx:227]: These routes are missing the `exact` prop they had in 4.0. I'm not sure if that was intended or something lost while cherry-picking the changes.

**Review by edmundoa — CHANGES_REQUESTED** [review]
I noticed there was a change in the Streams page and the link to "Manage Outputs" now moved to a menu. The link does not work, but we should hide the entry as well. Other than that it seems to do what it should 👍 

Screenshot 2020-11-25 at 11 46 21

File: graylog2-web-interface/src/routing/AppRouter.jsx (hunk 1 above)
Comment (Gaya) [reply graylog2-web-interface/src/routing/AppRouter.jsx:227]: That's a good one! I'll add them back

**Comment by Gaya** [comment]
> I noticed there was a change in the Streams page and the link to "Manage Outputs" now moved to a menu. The link does not work, but we should hide the entry as well.

Will change! Thanks for letting me know.

**Comment by edmundoa** [comment]
@Gaya I just merged #9537 in case you want to put the change in here as well. I think that's the last pending PR that was based on `cloud-3.3`.

**Comment by Gaya** [comment]
> @Gaya I just merged #9537 in case you want to put the change in here as well. I think that's the last pending PR that was based on `cloud-3.3`.

Awesome! @edmundoa I'll add that one in here as well. Thanks!

**Review by edmundoa — APPROVED** [review]
Looks good and works for me 👍

**Pull request merged**